src/domain/organizations/organizations.repository.ts

```diff
@@ -41,7 +41,7 @@ export class OrganizationsRepository implements IOrganizationsRepository {
 
     // @todo Move to UserOrganizationsRepository
     const userOrganization = new UserOrganization();
-    userOrganization.name = args.name;
+    userOrganization.name = `${organization.name} creator`;
```

Comment:
I'm not against this but do you think we should be handling internationalisation somehow?

Reply:
To my understanding, internationalization is not on the table right now, but indeed we should adapt this if the supported clients does internationalize their UIs 👍